core/templatetags/tests/test_social_tags.py

```diff
@@ -0,0 +1,39 @@
+from core.templatetags import social_tags
+
+
+def test_build_social_links():
+    actual = social_tags.build_twitter_link(
+        'Welcome to the UK',
+        'http://exred.trade.great')
+    expected = (
+        'https://twitter.com/intent/tweet?text=Welcome%20to%20the%20UK%20'
+        'http://exred.trade.great')
+    assert(actual == expected)
```

Comment:
assert is not a function - so for consistent style can you please change this to:

```
assert actual == expected
```